README.md

```diff
@@ -67,6 +67,10 @@ export ARTIFACTORY_TOKEN="my-token"
 The token will be used as the `Authorization` header with the value `Bearer
 <TOKEN>`.
 
+## Custom Certificate Authorities for Container Deployment
+
+If your artifactory server or extension download location is on a domain not signed by a default CA, then you will need to add those files either by volume mount or `docker cp` and then run `update-ca-certificates`.
```

Comment:
Minor formatting suggestion (just some capitalization to match the other headings, wrapping, and one more heading level).

```suggestion
### Custom certificate authorities for container deployment

If your Artifactory server or extension download location is on a domain not
signed by a default CA, then you will need to add those files either by volume
mount or `docker cp` and then run `update-ca-certificates`.
```